### DISCUSSION TITLE
Prolem setting up netbox container

### DISCUSSION THREAD

**Discussion by misch42** [netbox-community/netbox-docker#1306] [General]
hi,

I have a new ubuntu 24.04 installation with git and docker-compose-v2 installed.

When I copy the docker-compose.override.yml.example to the docker-compose.override.yml and start a docker compose pull, i get the following error:

netbox-worker Error
(...)
Error response from daemon: manifest for netboxcommunity/netbox:v4.1-3.0.0 not found: manifest unknown: manifest unknown

Any ideas?

**Comment by CrawliestCotter** [comment]
I'm getting the same error on fedora 40 with the following version of containerd, docker, and docker-compose:

\# containerd --version
containerd containerd.io 1.7.21 472731909fa34bd7bc9c087e4c27943f9835f111
\# docker --version
Docker version 27.2.0, build 3ab4256
\# docker compose version
Docker Compose version v2.29.2

\# docker compose pull
[+] Pulling 6/6
 ✔ netbox-worker Skipped - Image is already being pulled by netbox                                                                                                                       0.0s 
 ✔ netbox-housekeeping Skipped - Image is already being pulled by netbox                                                                                                                 0.0s 
 ✔ redis-cache Skipped - Image is already being pulled by redis                                                                                                                          0.0s 
 ✔ redis Pulled                                                                                                                                                                          0.9s 
 ✘ netbox Error                                                          manifest for netboxcommunity/netbox:v4.1-3.0.0 not found: manifest unknown: manifest unknown                    0.9s 
 ✔ postgres Pulled                                                                                                                                                                       0.9s 
Error response from daemon: manifest for netboxcommunity/netbox:v4.1-3.0.0 not found: manifest unknown: manifest unknown

\# cat docker-compose.override.yml
`services:`
`  netbox:`
`    ports:`
`      - "8000:8080"`

\# cat os-release
NAME="Fedora Linux"
VERSION="40 (Forty)"
...

**Comment by Netboxnoob** [comment]
## Same issue here....

```
root@netbox:/netbox-docker# docker-compose pull
Pulling postgres            ... done
Pulling redis               ... done
Pulling redis-cache         ... done
Pulling netbox              ... error
Pulling netbox-housekeeping ... error
Pulling netbox-worker       ... error

ERROR: for netbox-housekeeping  manifest for netboxcommunity/netbox:v4.1.0 not found: manifest unknown: manifest unknown

ERROR: for netbox  manifest for netboxcommunity/netbox:v4.1.0 not found: manifest unknown: manifest unknown

ERROR: for netbox-worker  manifest for netboxcommunity/netbox:v4.1.0 not found: manifest unknown: manifest unknown
ERROR: manifest for netboxcommunity/netbox:v4.1.0 not found: manifest unknown: manifest unknown
manifest for netboxcommunity/netbox:v4.1.0 not found: manifest unknown: manifest unknown
manifest for netboxcommunity/netbox:v4.1.0 not found: manifest unknown: manifest unknow
```

Does anyone know how to resolve this?

**Comment by elixdreamer** [comment]
Please use v4.0-2.9.1 until the issues have been resolved in v4.1-3.0.0.
Not sure if 'latest' is working too

**Comment by CrawliestCotter** [comment]
Downloading from the release page worked without issue. Might be a good idea to change the documentation in the "quick start" and "getting started" guides to pull from the release folder instead of the releases branch.

ie change:
git clone -b release https://github.com/netbox-community/netbox-docker.git
to
wget https://github.com/netbox-community/netbox-docker/archive/refs/tags/x.x.x.tar.gz
tar -xf x.x.x.tar.gz

I'm not sure if pulling from releases branch still causes an issue, but it seems to make more sense to download from the release folder anyway.
